src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -81,7 +82,9 @@
         BooleanParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "booleanParam"
         StringParameterDefinition.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "string"
         BuildDiscarderProperty.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "buildDiscarder"
-        LogRotator.LRDescriptor.class.isAnnotationPresent(Symbol.class); // "logRotator"
+        LogRotator.LRDescriptor.class.isAnnotationPresent(Symbol.class) && // "logRotator"
+        TimerTrigger.DescriptorImpl.class.isAnnotationPresent(Symbol.class) && // "disableConcurrentBuilds"
+        PipelineTriggersJobProperty.DescriptorImpl.class.isAnnotationPresent(Symbol.class); // "pipelineTriggers"
```

Comment:
:bug: this last clause is unconditionally true, since we declare that dependency. This constant is about symbols present in _core_.